docs/blog/newsletter-march-2023.rst

```diff
@@ -0,0 +1,110 @@
+
+Others mentioned having a localization voiceover gig to demonstrate using a chainsaw, working on contracts for a company that built and sold childcare facilities, writing instructions for a reading tracking software company, authoring an operations manual for a robotic systems defense contractor, and creating content for a bespoke application for the oil and gas industry. From the variety of examples, it can be said that documentarians are tasked with complex and exciting projects, even if it's just helping a parent write and edit books on sleight-of-hand magic.
+
+The most interesting projects that documentarians have worked on showcase the diverse range of topics and industries they can adapt to and the complexity of the information they need to communicate. As technology continues to evolve and shape our world, documentarians will remain a crucial part of the process, ensuring that information is shared effectively and efficiently with a broad audience. It's clear that it's already a fascinating and challenging field, and it’s expected that more innovative and interesting projects will emerge in the future.
+
+---------------------
+AI for documentarians
+---------------------
+
+Artificial intelligence (AI) is still top of mind for many. Based on a discussion this month, here's what the community thinks we should we be looking out for.
+
+First up, if you're working on a machine learning (ML) tool, keep an eye on data lineage. It's important to distinguish between the known input for the ML tool and These analysis or predictions that the tool outputs. For example, this might mean labelling a prediction on a person's age as "Age analysis result" rather than "Age of subject". If you're interested, the following article present more on the subject: `Always Leave An Analysis Paper Trail <https://counting.substack.com/p/data-science-practice-101-always-leave-an-analysis-paper-trail-cc17079fae5a>`__, and `An Introduction to the Data Biography <https://weallcount.com/2019/01/21/an-introduction-to-the-data-biography/>`__.
+
+A related challenge can be explaining to end users what AI is actually doing. One discussion participant had been interested in the `End-User-Centered Explainable AI Framework <https://weina.me/euca/>`__, a prototyping tool for explaining AI to users; another linked to an example of a documentation standard trying to be transparent about AI, IBM's `AI FactSheets 360 <https://aifs360.mybluemix.net/introduction>`__.
+
+The community was also interested in the ethics of AI. They recommended a few pieces:
+
+- A paper called “On the Dangers of Stochastic Parrots”.
```

Comment:
Is the link missing for this? Or maybe there is a reason not to include it.

Reply:
```suggestion
- A paper called `"On the Dangers of Stochastic Parrots" <https://dl.acm.org/doi/10.1145/3442188.3445922>`__.
```